### PR TITLE
Added note on Banana Cake Pop

### PR DESCRIPTION
**Pull request by yorek**
## Why make this change?

Added note on Banana Cake Pop usage

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,10 @@ If you want to propose a completely new feature, please create an RFC item. Good
 
 **Data API builder for Azure Databases** is licensed under the MIT license. See the [LICENSE](./LICENSE.txt) file for more details.
 
+## Third-Party Component Notice
+
+If you use the third-party software Banana Cake Pop by ChilliCream, Inc., as part of your Data API builder, you may have the option to save your work on ChilliCream's cloud service, via your separate ChilliCream account.  Microsoft is not affiliated with and does not endorse ChilliCream's service, and you use those services at your own option
```
Comment (Aniruddh25) [inline README.md:83]:
```suggestion
If you use the third-party software Banana Cake Pop by ChilliCream, Inc., as part of your Data API builder, you may have the option to save your work on ChilliCream's cloud service, via your separate ChilliCream account.  Microsoft is not affiliated with and does not endorse ChilliCream's service, and you use those services at your own option.
```

**Review by Aniruddh25 — APPROVED** [review]
LGTM, thanks for adding this!

File: README.md [hunk 2]
```diff
@@ -78,6 +78,10 @@ If you want to propose a completely new feature, please create an RFC item. Good
 
 **Data API builder for Azure Databases** is licensed under the MIT license. See the [LICENSE](./LICENSE.txt) file for more details.
 
+## Third-Party Component Notice
+
+If you use the third-party software Banana Cake Pop by ChilliCream, Inc., as part of your Data API builder, you may have the option to save your work on ChilliCream's cloud service, via your separate ChilliCream account.  Microsoft is not affiliated with and does not endorse ChilliCream's service, and you use those services at your own option.
```
Comment (abhishekkumams) [inline README.md:83]: should it be like `at your own risk` rather than `at your own option`

Comment (seantleonard) [inline README.md:83]:
```suggestion
If you use the third-party software Banana Cake Pop by ChilliCream, Inc., as part of your Data API builder solution, you may have the option to save your work on ChilliCream's cloud service, via your separate ChilliCream account.  Microsoft is not affiliated with and does not endorse ChilliCream's service, and you use those services at your own option.
```
->as part of your Data API builder solution?

Comment (yorek) [reply README.md:83]: That's what CELA wrote, I would leave it like they wrote :)

**Review by seantleonard — APPROVED** [review]

**Pull request merged**